src/Console/CommandRunner.php

```diff
@@ -320,8 +320,12 @@ protected function runCommand(CommandInterface $command, array $argv, ConsoleIo
     {
         try {
             $eventManager = $this->getEventManager();
-            $eventManager = $this->app->pluginEvents($eventManager);
-            $eventManager = $this->app->events($eventManager);
+            if (method_exists($this->app, 'pluginEvents')) {
+                $eventManager = $this->app->pluginEvents($eventManager);
+            }
+            if (method_exists($this->app, 'events')) {
+                $eventManager = $this->app->events($eventManager);
+            }
             $this->setEventManager($eventManager);
```

Comment:
By my testing this also can't be done in console only applications since this method asserts that `$this->app` is an instance of `EventDispatcherInterface` which `ConsoleApplicationInterface` are not